src/main/java/net/sf/jabref/logic/cleanup/BiblatexCleanup.java

```diff
@@ -25,33 +26,22 @@
             entry.getField(oldFieldName).ifPresent(oldValue -> {
                 if (!oldValue.isEmpty() && (!entry.getField(newFieldName).isPresent())) {
                     // There is content in the old field and no value in the new, so just copy
-                    entry.setField(newFieldName, oldValue);
-                    changes.add(new FieldChange(entry, newFieldName, null, oldValue));
-
-                    entry.clearField(oldFieldName);
-                    changes.add(new FieldChange(entry, oldFieldName, oldValue, null));
+                    changes.add(entry.setField(newFieldName, oldValue).orElse(null));
```

Comment:
entry.setField(...).ifPresent(changes::add), otherwise you add null values to the list which is not so nice.